src/App.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useState } from "react";
+import Edit from "./Image/edit.png";
+import monthName from "./Components/Monthname";
+import { ButtonEdit, Button } from "./StyleComponents/Button";
+import Clock from "react-live-clock";
+import Footer from "./StyleComponents/Footer";
+import { allUpdateNote } from "./Type/Type";
+import GlobalStyle from "./globalStyle";
+import {
+  Border,
+  ButtomBorder,
+  CardDiv,
+  CenterDiv,
+  CompletedCard,
+  NoteContainer,
+} from "./StyleComponents/NoteDiv";
+
+const App = () => {
+  const date = new Date();
+  const [note, setNote] = useState<string>("");
+  const [allNotes, setAllNotes] = useState<allUpdateNote[]>([
+    { id: 0, title: "", update: false },
+  ]);
+
+  const [checkUpdate, setCheckUpdate] = useState(false);
+  const [updateIndex, setUpdateIndex] = useState(0);
+
+  const NoteList = allNotes.filter((value) => {
+    return value.update === false;
+  });
+
+  const completeList = allNotes.filter((value) => {
+    return value.update === true;
+  });
+
+  const updateNote = (updateNoteIndex: number) => {
+    setNote(allNotes[updateNoteIndex].title);
+    setCheckUpdate(true);
+    setUpdateIndex(updateNoteIndex);
+  };
+
+  const inputData = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setNote(event.target.value);
+  };
+
+  const dispalyNote = (note: string) => {
+    if (note === "" || note.trim() === "") {
+      alert("content Add");
+    } else if (checkUpdate == true) {
+      const updatedTodoListArray = [...allNotes];
+      updatedTodoListArray[updateIndex].title = note;
+      setAllNotes(updatedTodoListArray);
+      setNote("");
+      setCheckUpdate(false);
+    } else {
+      setAllNotes([
+        ...allNotes,
+        {
+          id: allNotes.length,
+          title: note.trim(),
+          update: false,
+        },
+      ]);
+    }
+    setNote("");
+  };
+
+  const deleteNote = (id: number) => {
+    setAllNotes((oldNote) => {
+      return oldNote.filter((element, index) => {
+        return element.id !== id;
+      });
+    });
+    setNote("");
+  };
+
+  const checkComplete = (id: number) => {
```

Comment:
What complete you are checking ??

Reply:
that the note is completed or not if completed than add that particular note in completed list